src/pdsparsemat.jl

```diff
@@ -84,22 +84,22 @@ end
 
 function X_A_Xt(a::PDSparseMat, x::StridedMatrix)
     z = x*sparse(chol_lower(a.chol))
-    A_mul_Bt(z, z)
+    z * z'
 end
 
 
 function Xt_A_X(a::PDSparseMat, x::StridedMatrix)
-    z = At_mul_B(x, sparse(chol_lower(a.chol)))
-    A_mul_Bt(z, z)
+    z = x' * sparse(chol_lower(a.chol))
+    z * z'
 end
 
 
 function X_invA_Xt(a::PDSparseMat, x::StridedMatrix)
-    z = a \ x'
+    z = a.chol \ collect(x')
```

Comment:
Why does `x` have to be materialized? Is there no appropriate solver method for whatever type `a.chol` is?

Reply:
No, there isn't an appropriate solver.